### DISCUSSION TITLE
Wegpunkt überspringen

### DISCUSSION THREAD

**Discussion by ToToGSARider** [devemux86/cruiser#212] [Q&A] [answered]
Wie überspringe ich eigentlich einen Wegpunkt bei deaktivierter Neuberechnung. Ich habe beim ersten Test angehalten und den Punkt einfach entfernt, hatte aber dann keine Route mehr. Ich möchte auch nicht zur Route Navigiert werden. Ich bin auf der Straße stehen geblieben und hätte eigentlich nach rechts weiter gemusst zum nächsten Zwischenziel. Ich habe dieses dann entfernt. Hätte ich es überhaupt entfernen müssen? Was passiert wenn ich an einem Zwischenziel einfach vorbeifahre und nicht dort halte und wieder auf die Route gehe.

**Comment by devemux86** [comment, marked as answer]
@ToToGSARider thanks for the question!

With Cruiser, you don't have to skip any waypoints! Cruiser has 2 navigation modes:
- **Normal navigation**: you can leave and re-enter the route anytime and anywhere, the navigation just continues.
- **Strict route following**: if you enable it, you will have to navigate the route passing all waypoints.

You can still "**Skip next waypoint**" or "**Skip many waypoints**" by pressing the navigation panel on the top-left (see images below).
Cruiser calculates the route only to the next waypoint, **your curvy route remains unchanged**.

---

@ToToGSARider danke für die Frage!

Mit Cruiser müssen Sie keine Wegpunkte auslassen! Cruiser hat 2 Navigationsmodi:
- **Normale Navigation**: Sie können die Route jederzeit und überall verlassen und wieder einsteigen, die Navigation läuft einfach weiter.
- **Strenge Routenverfolgung**: Wenn Sie diese Funktion aktivieren, müssen Sie die Route an allen Zwischenzielen vorbei navigieren.

Sie können immer noch den "**nächsten Wegpunkt überspringen**" oder "**viele Wegpunkte überspringen**", indem Sie auf das Navigationsfeld oben links drücken (siehe Bilder unten).
Cruiser berechnet die Route nur bis zum nächsten Wegpunkt, **Ihre kurvenreiche Route bleibt unverändert**.

<img src="https://user-images.githubusercontent.com/3484020/222767346-f304b504-3ce4-412d-b7ab-bd5f045e20a1.jpg" width="50%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/13ce6be7-c710-49ca-ae35-f3fc3d9913cd" width="50%"/>

**Reply by ToToGSARider** [reply]
@devemux86 Könnte man bei der Strikten Routenverfolgung nicht auch noch das WunderlinQ mit einbeziehen? Kurzer Druck nach rechts öffnet Menü, hoch und runter Rollen ist die Auswahl, langer Druck bestätigen

@devemux86 Couldn't the WunderlinQ also be included in the Strict Route Tracking? Short press right opens menu, scroll up and down is the selection, long press confirm

**Reply by devemux86** [reply]
@ToToGSARider

Thanks for the interesting idea!
I could set up the menu to open externally, but the Android screens work with touch events.
Then we need select the waypoint by pressing on it?

Danke für die interessante Idee!
Ich könnte das Menü so einrichten, dass es extern geöffnet wird, die Android-Bildschirme funktionieren jedoch mit Berührungsereignissen.
Dann müssen wir den Wegpunkt auswählen, indem wir darauf drücken?

cc @blackboxembedded

**Reply by ToToGSARider** [reply]
@devemux86 cc @blackboxembedded 
Kurzer druck nach rechts öffnet das Menü; durch hoch und runter rollen werden die Schaltflächen / Wegpunkte angewählt; durch langen druck rechts wird die Auswahl bestätigt. Bei den Aktionen gibt es ja den Punkt Übersicht. Hier müßte dann auch noch eine Auswahl kommen Übersicht beenden. Man müßte sonst Navigation beenden und wieder starten. Die Übersicht kann man ja auch verwenden wenn keine Strickte Navigation läuft. 

@devemux86 cc @blackboxembedded 
Short press to the right opens the menu; scroll up and down to select the buttons / waypoints; long press to the right confirms the selection. With the actions there is the point overview. Here should also come a selection End overview. Otherwise you would have to end navigation and start it again. The overview can also be used when no strict navigation is running. 

Translated with www.DeepL.com/Translator (free version)

**Reply by devemux86** [reply]
@ToToGSARider 

You do not have to restart the navigation.
The route overview is displayed for a few seconds, then it automatically returns to normal navigation.
You can also press the **location button** to return to normal navigation.

Sie müssen die Navigation nicht neu starten.
Die Routenübersicht wird einige Sekunden lang angezeigt, dann kehrt sie automatisch zur normalen Navigation zurück.
Sie können auch die **Standorttaste** drücken, um zur normalen Navigation zurückzukehren.

**Reply by ToToGSARider** [reply]
@devemux86 ah Okay, das wußte ich nicht. Ich wußte nicht einmal, dass oben dieses Menü geöffnet wird. Gab es das in der K2 App auch schon? Ich möchte so viel wie möglich mit meinem Kontrollrad an der BMW in Verbindung mit dem WunderlinQ machen.

@devemux86 ah Okay, I didn't know that. I didn't even know that this menu opened at the top. Did that exist in the K2 app as well? I want to do as much as possible with my control wheel on the BMW in conjunction with the WunderlinQ.

**Reply by devemux86** [reply]
> Gab es das in der K2 App auch schon?

No, Cruiser has much more additional new features than K1 / K2. :slightly_smiling_face: 

Nein, Cruiser hat viel mehr zusätzliche neue Funktionen als K1 / K2. :slightly_smiling_face:

**Reply by ToToGSARider** [reply]
@devemux86 Guten Morgen, gibt es schon was neues von Keith zum Thema Umsetzung der Idee mit der weiteren Bedienung der App über das WunderlinQ. Ich hätte da ja noch ne andere verrückte Idee. Evtl. mache ich dazu später mal ein Thema auf. 

@devemux86 Good morning, is there already something new from Keith about the implementation of the idea with the further operation of the app via the WunderlinQ. I would have there still ne other crazy idea. Maybe I'll open a topic about it later.

**Reply by devemux86** [reply]
Hi @ToToGSARider,

cc @blackboxembedded

Do you mean to scroll in waypoints list and select a waypoint with the WunderLINQ controller?
There has to be support in Cruiser for such actions, then can use available WunderLINQ keys.
Currently with WunderLINQ we can zoom or move the map.

Meinen Sie das Blättern in der Wegpunktliste und die Auswahl eines Wegpunktes mit dem WunderLINQ-Controller?
Es muss im Cruiser eine Unterstützung für solche Aktionen geben, dann können die verfügbaren WunderLINQ-Tasten verwendet werden.
Derzeit können wir mit WunderLINQ die Karte zoomen oder verschieben.

**Reply by ToToGSARider** [reply]
@devemux86 ich meinte die Funktion während der Navigation. Navigation läuft, dann rücke ich mit dem Finger links oben auf nächste Aktion und dann geht der Balken auf mit möglichen Aktionen. Jetzt wäre es cool, wenn man mit dem Kontrollrad hoch und runter Rollen könnte um eine Aktion wie zum Beispiel Übersicht auswählen könnte oder überspringen. Das geht ja auch in der Navigation ohne strickte Navigation. Das öffnen der Funktion, was man jetzt mit dem Finger macht, könnte man ja auslösen durch druck nach rechts auf dem Kontrollrad.

@devemux86 I meant the function during navigation. Navigation is running, then I move with my finger on the top left to next action and then the bar goes up with possible actions. Now it would be cool if you could scroll up and down with the control wheel to select an action such as overview or skip. This is also possible in the navigation without strict navigation. The opening of the function, which is now done with the finger, could be triggered by pressing right on the control wheel.

**Reply by devemux86** [reply]
@ToToGSARider thanks for the explanation. This needs to be supported in Cruiser first, then we could add the connection with WunderLINQ.

@ToToGSARider danke für die Erklärung. Dies muss zuerst in Cruiser unterstützt werden, dann könnten wir die Verbindung mit WunderLINQ hinzufügen.

**Reply by ToToGSARider** [reply]
@devemux86 kein Problem, wenn ich helfen kann, helfe ich sehr gerne :-) 

@devemux86 no problem, if I can help I am very happy to help :-)

**Reply by ToToGSARider** [reply]
@devemux86  gibt es hier eigentlich schon was neues zum Thema? Sind weitere Funktionen aus der WunderLINQ Steuerung im Cruiser umgesetzt worden. 

@devemux86  Is there anything new on the subject here yet? Have further functions from the WunderLINQ control system been implemented in the Cruiser?

**Reply by devemux86** [reply]
@ToToGSARider

WunderLINQ currently supports zoom and scroll in Cruiser. More functions are in my list for implementation.

WunderLINQ unterstützt derzeit Zoom und Scrollen in Cruiser. Weitere Funktionen stehen auf meiner Liste zur Implementierung.

**Reply by ToToGSARider** [reply]
@devemux86 Okay, dann warte ich mal ab, bis hier dann wieder eine Antwort steht. 

@devemux86 Okay, then I'll wait until there's an answer here again.

**Reply by ToToGSARider** [reply]
@devemux86 welche Funktionen sind denn noch so geplant in der Zukunft? 

@devemux86 What other functions are planned for the future?

**Reply by devemux86** [reply]
@ToToGSARider 

Here are some of the planned functions that you can already test:

Hier finden Sie einige der geplanten Funktionen, die Sie bereits testen können:

- https://github.com/devemux86/cruiser/discussions/544

- https://github.com/devemux86/cruiser/discussions/559

**Reply by ToToGSARider** [reply]
@devemux86 
Wenn ich mir die Beta Cruiser 5 lade, bekommen ich dann automatisch auch wieder die aktuellen Versionen, wenn Cruiser 5 dann im Playstore ist? 
Was sind die neuen Funktionen im Cruiser 5.0 für das WunderLINQ? das konnte ich jetzt nicht so genau erkennen.
Wann kommt Cruiser 5.0?
@devemux86 
If I download the Cruiser 5 beta, will I automatically get the latest versions when Cruiser 5 is in the Playstore? 
What are the new functions in Cruiser 5.0 for the WunderLINQ?
When will Cruiser 5.0 be released?

**Reply by devemux86** [reply]
Cruiser 5 is not yet available for testing. We will see what is possible with WunderLINQ.
Beta versions are always updated from the releases on Google Play.

Cruiser 5 ist noch nicht zum Testen verfügbar. Wir werden sehen, was mit WunderLINQ möglich ist.
Beta-Versionen werden immer von den Veröffentlichungen auf Google Play aktualisiert.

**Comment by devemux86** [comment]
@ToToGSARider 

Remember to select "BRouter (offline)" in the routing settings and set up the BRouter, see instructions:

Denken Sie daran, in den Routing-Einstellungen "BRouter (offline)" auszuwählen und den BRouter einzurichten, siehe Anleitung:

- https://github.com/devemux86/cruiser/discussions/137

**Reply by ToToGSARider** [reply]
@devemux86 muss ich den BROUTER unbedingt einrichten? Ich habe es gemerkt und mich wieder erinnert wie es bei K2 war, einfach am Zwischenziel vorbei fahren und ignorieren. Ich plane eigentlich immer alle Touren in Kurviger und exportiere diese in meine Magenta Cloud. Bei Baustellen zoome ich mit WunderlinQ aus der Karte und schaue wo ich wieder auf die Route komme. Neuberechnung ist immer aus.


@devemux86 do I really need to set up the BROUTER? I noticed and remembered again how it was with K2, just drive past the waypoint and ignore. I actually always plan all trips in Kurviger and export them to my Magenta Cloud. At road works I zoom out of the map with WunderlinQ and look where I get back on the route. Recalculation is always off.

**Reply by devemux86** [reply]
@ToToGSARider 

BRouter (offline) is only needed if you want to re-calculate part of the route offline.
Otherwise you can simply leave the route and re-enter it at any point!

BRouter (offline) wird nur benötigt, wenn Sie einen Teil der Route offline neu berechnen wollen.
Ansonsten können Sie die Route einfach verlassen und an beliebiger Stelle neu eingeben!
